google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java

```diff
@@ -87,6 +87,12 @@
         required = true,
         description = "The ID of a store sales conversion action")
     private Long conversionActionId;
+    
+    @Parameter(
+        names = ArgumentNames.CUSTOM_KEY,
+        required = false,
+        description = "Only required if uploading data with custom key and values")
```

Comment:
Can we add a bit more detail here to explain what a custom key/value pair is, when it's required and why please?

Reply:
Thanks nwbirnie. I added an expanded explanation.